<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Proyecto de Sistemas Operativos</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>831</width>
      <height>391</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Informacion del Sistema</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>131</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>351</width>
        <height>121</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>349</width>
         <height>119</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="labelDdr">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>40</y>
        <width>191</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollAreaDdr">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>40</y>
        <width>451</width>
        <height>121</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>449</width>
         <height>119</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>KacstBook</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Datos de Memoria</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>40</y>
        <width>131</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>20</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>KacstBook</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Datos de la Red</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelDdr2">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>40</y>
        <width>221</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
      </property>
     </widget>
     <widget class="QLabel" name="labelDdd">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>121</width>
        <height>151</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollAreaDdd">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>121</width>
        <height>151</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>119</width>
         <height>149</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>160</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>KacstBook</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Datos del Disco</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelDds">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>190</y>
        <width>141</width>
        <height>151</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="QLabel" name="labelDds2">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>190</y>
        <width>211</width>
        <height>151</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollAreaDds">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>190</y>
        <width>421</width>
        <height>151</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>419</width>
         <height>149</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>170</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>KacstBook</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Datos del Sistema</string>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollAreaDisco2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>190</y>
        <width>120</width>
        <height>151</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>118</width>
         <height>149</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="labelDisco2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>190</y>
        <width>121</width>
        <height>151</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="labelDisco3">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>190</y>
        <width>121</width>
        <height>151</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollAreaDd3">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>190</y>
        <width>121</width>
        <height>151</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_6">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>119</width>
         <height>149</height>
        </rect>
       </property>
      </widget>
     </widget>
     <zorder>label</zorder>
     <zorder>scrollArea</zorder>
     <zorder>label_3</zorder>
     <zorder>label_2</zorder>
     <zorder>label_4</zorder>
     <zorder>labelDdd</zorder>
     <zorder>scrollAreaDdd</zorder>
     <zorder>label_5</zorder>
     <zorder>labelDds</zorder>
     <zorder>scrollAreaDds</zorder>
     <zorder>label_6</zorder>
     <zorder>labelDds2</zorder>
     <zorder>scrollAreaDdr</zorder>
     <zorder>labelDdr</zorder>
     <zorder>labelDdr2</zorder>
     <zorder>scrollAreaDisco2</zorder>
     <zorder>labelDisco2</zorder>
     <zorder>labelDisco3</zorder>
     <zorder>scrollAreaDd3</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Lanzador de Aplicaciones</string>
     </attribute>
     <widget class="QLineEdit" name="lineEditLanzar">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>110</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>90</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>KacstBook</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Lanzar Aplicacion</string>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="commandLinkButtonIni">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>110</y>
        <width>174</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Iniciar</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditPID">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>190</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="commandLinkButtonPID">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>190</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Obtener PID</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>170</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>KacstBook</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Conocer PID de un proceso</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditKill">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>270</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="commandLinkButtonKill">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>270</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Terminar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>250</y>
        <width>231</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>KacstBook</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Terminar aplicacion (Ingrese PID)</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Acerca de</string>
     </attribute>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>90</y>
        <width>191</width>
        <height>191</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="Imagenes.qrc">:/Unet.png</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>KacstBook</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Programa Realizado por:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>KacstBook</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Henry Bravo</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>KacstBook</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>henry.bravo@unet.edu.ve</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>40</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>KacstBook</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Jorge Gomez</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>290</y>
        <width>411</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>KacstBook</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Programa realizado para la materia de Sistemas Operativos</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>310</y>
        <width>411</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>KacstBook</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>                          de la carrera Ingenieria en Informatica</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>70</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>KacstBook</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>jorgew.gomez@unet.edu.ve</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="labelSO">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>-8</y>
      <width>331</width>
      <height>39</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>KacstBook</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Imagenes.qrc"/>
 </resources>
 <connections/>
</ui>
